build/android/patches/irrlicht-back_button.patch

```diff
@@ -1,10 +1,11 @@
 --- irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp	2014-06-03 20:56:21.289559503 +0200
 +++ irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp.orig	2014-06-03 20:57:39.281556749 +0200
-@@ -423,6 +423,7 @@
+@@ -486,7 +488,7 @@
+ 				event.KeyInput.Char = 0;
  			}
- 			
- 			device->postEventFromUser(event);
-+			status = 1;
+ 
+-			device->postEventFromUser(event);
++			status = device->postEventFromUser(event) ? 1 : 0;

```

Comment:
Does the C++ standard define how bool gets stored? status **must** be 0 if the return value is false, and **must** be 1 if the return value is true, no other values are allowed.

Are you proposing a cast? Or does c++ cast it automatically